File: server/read/fastify/src/models/BoardSimpleReadData.ts
import { Board } from '../entities/Board';

export class BoardSimpleReadData {
  uuid: string;
  uniqueId: string;
  name: string;
  description: string;

  constructor(board: Board) {
    this.uuid = board.uuid;
    this.uniqueId = board.uniqueId;
    this.name = board.name;
    this.description = board.description;
  }
}

File: server/write/spring/src/main/kotlin/com/board/server/write/spring/models/BoardSimpleReadData.kt
package com.board.server.write.spring.models

import com.board.server.write.spring.entities.BoardEntity
import java.util.*

data class BoardSimpleReadData(private val board: BoardEntity) {
    val uuid: UUID = board.uuid
    val uniqueId: String = board.uniqueId
    val name: String = board.name
}
